cmd/helm/install_test.go

```diff
@@ -111,9 +111,9 @@ func TestInstall(t *testing.T) {
 		{
 			name:     "install with name-template",
 			args:     []string{"testdata/testcharts/alpine"},
-			flags:    []string{"--name-template", "{{upper \"foobar\"}}"},
-			expected: "FOOBAR",
-			resp:     helm.ReleaseMock(&helm.MockReleaseOptions{Name: "FOOBAR"}),
+			flags:    []string{"--name-template", "{{lower \"FOOBAR\"}}"},
```

Comment:
Would you mind adding failure cases to the bottom here? It's important that we both capture good test cases as well as the bad ones rather than removing them entirely.

```
		// Install, using a bad release name
		{
			name:  "install chart with release name using capitals",
			args:  []string{"testdata/testcharts/alpine"},
			flags: []string{"--name", "FOO"},
			err:   true,
		},
		{
			name:  "install chart with release name using periods",
			args:  []string{"testdata/testcharts/alpine"},
			flags: []string{"--name", "foo.bar"},
			err:   true,
		},
		{
			name:  "install chart with release name using underscores",
			args:  []string{"testdata/testcharts/alpine"},
			flags: []string{"--name", "foo_bar"},
			err:   true,
		},
		// Install, using a bad name-template
		{
			name:  "install with name-template",
			args:  []string{"testdata/testcharts/alpine"},
			flags: []string{"--name-template", "{{UPPER \"foobar\"}}"},
			err:   true,
		},
```

Reply:
thanks for your advice. already added them.